### PR TITLE
use go 1.13 and rely on os.UserConfigDir to find config location

### PR DESCRIPTION
**Pull request by pyr**
This should fix failures to find the config file's location on
Windows. As per a recommendation from @greut, thanks!

Somewhat related to #197 though it might not fix the issue.

### CODE REVIEW

**Comment by pyr** [comment]
Naive fix attempt, builds locally. I don't know the implications of requiring go 1.13 though.

**Comment by pyr** [comment]
@horakg can you try a build off of this branch to see if the behavior changes?

File: cmd/root.go [hunk 1]
```diff
@@ -200,18 +200,12 @@ func initConfig() {
...
-		if found {
-			gConfigFolder = path.Join(home, ".config", "exoscale")
-		} else {
-			// The XDG spec specifies a default XDG_CONFIG_HOME in $HOME/.config
-			gConfigFolder = path.Join(usr.HomeDir, ".config", "exoscale")
-		}
+	cfgdir, err := os.UserConfigDir()
+	if err != nil {
+		log.Fatal(fmt.Errorf("could not find configuration directory: %s", err))
+		return
```
Comment (7fELF) [inline cmd/root.go]:
```suggestion
```
[`log.Fatal`](https://godoc.org/log#Logger.Fatal) calls calls `os.Exit(1)`, no need to return

File: cmd/root.go [hunk 2]
```diff
@@ -200,18 +200,12 @@ func initConfig() {
...
-		home, found := os.LookupEnv("HOME")
-		if found {
-			gConfigFolder = path.Join(home, ".config", "exoscale")
-		} else {
-			// The XDG spec specifies a default XDG_CONFIG_HOME in $HOME/.config
-			gConfigFolder = path.Join(usr.HomeDir, ".config", "exoscale")
-		}
+	cfgdir, err := os.UserConfigDir()
+	if err != nil {
+		log.Fatal(fmt.Errorf("could not find configuration directory: %s", err))
```
Comment (7fELF) [inline cmd/root.go]:
```suggestion
		log.Fatalf("could not find configuration directory: %s", err)
```

Comment (horakg) [reply cmd/root.go]: @pyr how can I build it for windows?

**Comment by falzm** [comment]
@horakg here is a Windows build of this branch: 
[exoscale-cli_v1.10.0-SNAPSHOT-7b2f70e_windows_amd64.zip](https://github.com/exoscale/cli/files/4151927/exoscale-cli_v1.10.0-SNAPSHOT-7b2f70e_windows_amd64.zip)

**Comment by horakg** [comment]
@falzm @pyr this version works on my office pc
Shall I also try it at home?

**Comment by falzm** [comment]
@horakg for belt and suspenders' sake I'd say yes: if confirmed, we can definitively put this problem behind :)

**Comment by horakg** [comment]
works also at home

**Comment by falzm** [comment]
Great, thanks for confirming.

**Pull request merged**